---
- id: ossindex:pypi:pyxdg:360949
  title: '[CVE-2014-1624]  Improper Link Resolution Before File Access ("Link Following")'
  description: Race condition in the xdg.BaseDirectory.get_runtime_dir function in
    python-xdg 0.25 allows local users to overwrite arbitrary files by pre-creating
    /tmp/pyxdg-runtime-dir-fallback-victim to point to a victim-owned location, then
    replacing it with a symlink to an attacker-controlled location once the get_runtime_dir
    function is called.
  affected_package: pyxdg
  vulnerable_versions:
  - '0.25'
  package_manager: pypi
  cve:
  - CVE-2014-1624
  disclosed_date: 2014-01-28
  created_date: 2014-01-28
  last_modified_date: 2017-04-12
  credit:
  - OSSIndex
  references:
  - http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=736247
  - http://www.openwall.com/lists/oss-security/2014/01/21/3
  - http://www.openwall.com/lists/oss-security/2014/01/21/4
  - http://www.securityfocus.com/bid/65042
  - http://xforce.iss.net/xforce/xfdb/90618
  - https://exchange.xforce.ibmcloud.com/vulnerabilities/90618
  source_url: https://ossindex.net/resource/package/8401249338/vulnerabilities
- id: snykio:pypi:pyxdg:40263
  title: Arbitrary File Overwrite
  description: |
    [`pyxdg`][1] is a PyXDG contains implementations of freedesktop.org
    standards in python. Race condition in the
    xdg.BaseDirectory.get\_runtime\_dir function in python-xdg 0.25 allows
    local users to overwrite arbitrary files by pre-creating
    /tmp/pyxdg-runtime-dir-fallback-victim to point to a victim-owned
    location, then replacing it with a symlink to an attacker-controlled
    location once the get\_runtime\_dir function is called.



    [1]: https://pypi.python.org/pypi/pyxdg
  affected_package: pyxdg
  vulnerable_versions:
  - ">= 0.25"
  severity: medium
  package_manager: pypi
  cve:
  - CVE-2014-1624
  cwe:
  - CWE-59
  disclosed_date: 2014-01-21
  created_date: 2014-01-21
  last_modified_date: 2014-01-21
  credit:
  - Unknown
  references:
  - http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=736247
  - http://www.openwall.com/lists/oss-security/2014/01/21/4
  - https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2014-1624
  source_url: https://snyk.io/vuln/SNYK-PYTHON-PYXDG-40263
